docs/user-guide/media-editor-window.md

```diff
@@ -0,0 +1,93 @@
+# Media Editor Window
+**********************
+
+The **Change media source** button allows to access the Media Editor Window:
+
+<img src="../img/media-editor-window/media-editor-window.jpg" class="ms-docimage"/>
+
+Through this window, you can add or edit three different types of media:
+
+* Images
+
+* Videos 
+
+* Maps
+
+## Images
+
+In order to add an image, the user can click on the **Images** button <img src="../img/button/images.jpg" class="ms-docbutton"/> for switch to the images section. In this section of the media editor window, with a click on the **Add** button <img src="../img/button/+++.jpg" class="ms-docbutton"/>, it is possible to define the image settings.
```

Comment:
In order to add an image, the stroy editor can click on the **Images** tab <img src="../img/button/images.jpg" class="ms-docbutton"/> in order to switch to the images section. In this section of the media editor window, with a click on the **Add** button <img src="../img/button/+++.jpg" class="ms-docbutton"/>, it is possible to define the image settings.